src/App.js

```diff
@@ -1,41 +1,22 @@
 import "./App.css";
-import logo from "./logo.png";
 
+import { Navbar } from "./components/components";
+import { Authentication, Browse, Cart, Home, Wishlist } from "./pages/pages";
+
+import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <BrowserRouter>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/Authentication" element={<Authentication />} />
+          <Route path="/Browse" element={<Browse />} />
+          <Route path="/Cart" element={<Cart />} />
+          <Route path="/Wishlist" element={<Wishlist />} />
```

Comment:
Lowercase letters should be used in path as url is always in lowercase.

Reply:
Done